plugins/source/aws/docs/event-based-sync.md

```diff
@@ -1,5 +1,5 @@
 :::callout{type="info"}
-This feature is currently in a closed preview. Sign up for access using this [form](https://cloudquery.typeform.com/to/Dvdn6P2u)
```

Comment:
Can we keep the sign up link? I don't see a reason to remove it yet. If someone signs up we can tell them we'll release the premium version soon

Reply:
d8876ca0b96fa0570b02712cc188a5f743681094